xmtp_mls/src/subscriptions.rs

```diff
@@ -236,8 +235,9 @@ where
     ) -> StreamHandle<Result<(), ClientError>> {
         let (tx, rx) = oneshot::channel();
 
-        let handle = tokio::spawn(async move {
-            let mut stream = client.stream_conversations().await?;
+        let handle = crate::spawn(async move {
+            let stream = client.stream_conversations().await?;
+            futures::pin_mut!(stream);
```

Comment:
this should be a minor performance improvement, streams are entirely stack-based, even in the `with_callback` methods.